src/util/compareThought.js

```diff
@@ -23,6 +23,18 @@ export const compare = (a, b) => a > b ? 1 : a < b ? -1 : 0
 //       )
 // }
 
+/** A comparator that sorts string starting with emojis ahead of other. */
+export const compareStringsWithEmojis = (a, b) => {
+  const aStartsWithEmoji = regexEmojis.test(a)
+  const bStartsWithEmoji = regexEmojis.test(b)
+  const aStripped = removeEmojisAndSpaces(a)
+  const bStripped = removeEmojisAndSpaces(b)
+  return aStartsWithEmoji && !bStartsWithEmoji ? -1
+    : bStartsWithEmoji && !aStartsWithEmoji ? 1
+    : aStartsWithEmoji && bStartsWithEmoji ? compareReasonable(aStripped, bStripped)
```

Comment:
You'll want to simply return `0` if they both start with an emoji. Each comparator functions independently and should only compare its relevant criterion. `makeOrderedComparator` handles composing the comparators together in the correct order.